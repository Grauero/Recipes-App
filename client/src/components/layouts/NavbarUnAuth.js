import React from 'react';
import { NavLink } from 'react-router-dom';

const NavbarUnAuth = () => (
  <ul>
    <li>
      <NavLink exact to="/">
        Home
      </NavLink>
    </li>
    <li>
      <NavLink to="/search">Search</NavLink>
    </li>
    <li>
      <NavLink to="/signin">Signin</NavLink>
    </li>
    <li>
      <NavLink to="/signup">Signup</NavLink>
    </li>
  </ul>
);

export default NavbarUnAuth;
