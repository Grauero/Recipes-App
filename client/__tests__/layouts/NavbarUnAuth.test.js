import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import NavbarUnAuth from '../../src/components/layouts/NavbarUnAuth';

let component;

beforeEach(() => {
  component = shallow(<NavbarUnAuth />);
});

afterEach(() => {
  component.unmount();
});

it('matches snapshot', () => {
  expect(toJSON(component)).toMatchSnapshot();
});
