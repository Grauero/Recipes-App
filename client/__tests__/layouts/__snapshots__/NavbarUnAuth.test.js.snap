// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<ul>
  <li>
    <NavLink
      activeClassName="active"
      aria-current="page"
      exact={true}
      to="/"
    >
      Home
    </NavLink>
  </li>
  <li>
    <NavLink
      activeClassName="active"
      aria-current="page"
      to="/search"
    >
      Search
    </NavLink>
  </li>
  <li>
    <NavLink
      activeClassName="active"
      aria-current="page"
      to="/signin"
    >
      Signin
    </NavLink>
  </li>
  <li>
    <NavLink
      activeClassName="active"
      aria-current="page"
      to="/signup"
    >
      Signup
    </NavLink>
  </li>
</ul>
`;
